src/App.js

```diff
@@ -121,36 +134,84 @@ export default class App extends Component {
     );
   }
 
+  /**
+   * Callback function to update "viewport" in state
+   * @param newViewport 
+   */
+  updateViewport = (newViewport) => {
+    const {viewport} = this.state;
+    this.setState({
+      viewport: {
+        ...viewport,
+        ...newViewport,
+      }
+    });
+  }
+
+  /**
+   * Returns a list of the county GeoJSON features
+   */
+  getIllinoisCountyFeatures = () => {
+    if (this.state.illinois_counties != null) {
+      var countyFeatures =  this.state.illinois_counties.features.map((feature) => {
+        return feature;
+      })
+      const sortedCountyFeatures = countyFeatures.sort((a,b) => (a.properties.NAME > b.properties.NAME) ? 1 : -1);
+
+      return sortedCountyFeatures;
+    } else {
```

Comment:
else is not necessary here - again just a cosmetic comment

Reply:
Changed